addons/bestja_project_hierarchy/models.py

```diff
@@ -43,6 +43,7 @@ class ProjectInvitation(models.Model):
         compute_sudo=True,
     )
     invited_projects = fields.One2many('bestja.project', inverse_name='parent_invitation')
+    inviter = fields.Boolean(string="Zapraszający", compute='_compute_inviter')
```

Comment:
The only change I want from you is the name change :) The other thing is weird only in a sense that Odoo is weird ;) So I just want the name to reflect that, because this is kinda hard to grasp.